src/components/ArchivePage/WorkshopInfoItem.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Chip, Typography, Link, ListItem } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+	listItem: {
+		display: 'flex',
+		flexDirection: 'column'
+	},
+	links: {
+		display: 'flex',
+		flexDirection: 'row',
+		flexWrap: 'wrap'
+	},
+	chips: {
+		display: 'flex',
+		alignItems: 'center',
+		flexWrap: 'wrap'
+	},
+	chip: {
+		margin: theme.spacing(0.125, 0.25, 0.125, 0)
+	}
+
+}));
+
+function LinkItem({ link, text, divider, ...props }) {
+	return <Typography>
+		{divider ? <span>{`· `}</span> : null }
+		<Link href={link} {...props} >{text}</Link>
+		&nbsp; {/* HTML code for space */}
```

Comment:
nbsp is not space; it's a [non-breaking space](https://en.wikipedia.org/wiki/Non-breaking_space), which prevents line breaks at this particular location. You may wish to check if this is the intended effect.